src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -165,13 +166,158 @@ public partial class UpdateDurableFunction
         }
     }
 
+    /// <summary>
+    /// Asynchronously updates list of instances in a study
+    /// </summary>
+    /// <remarks>
+    /// Durable functions are reliable, and their implementations will be executed repeatedly over the lifetime of
+    /// a single instance.
+    /// </remarks>
+    /// <param name="context">The context for the orchestration instance.</param>
+    /// <param name="logger">A diagnostic logger.</param>
+    /// <returns>A task representing the <see cref="UpdateInstancesV3Async"/> operation.</returns>
+    /// <exception cref="ArgumentNullException">
+    /// <paramref name="context"/> or <paramref name="logger"/> is <see langword="null"/>.
+    /// </exception>
+    /// <exception cref="FormatException">Orchestration instance ID is invalid.</exception>
+    [FunctionName(nameof(UpdateInstancesV3Async))]
+    public async Task UpdateInstancesV3Async(
```

Comment:
Is the only difference the calls to v3 activities?

Reply:
Oh, I'm not going to ask that lolol (don't fret). There was just a lot of "new" code and I wanted to confirm that the differences are pretty insignificant